src/service/iot_config.proto

```diff
@@ -653,22 +659,18 @@ service route {
   rpc update_devaddr_ranges(stream route_update_devaddr_ranges_req_v1)
       returns (route_devaddr_ranges_res_v1);
 
-  // Stream Routes update (auth admin only)
-  rpc stream(route_stream_req_v1) returns (stream route_stream_res_v1);
-}
+  // Session Key Filters (aka SKFs)
 
-service session_key_filter {
-  // List Filters for an Org (auth delegate_keys/owner/admin)
-  rpc list(session_key_filter_list_req_v1)
-      returns (stream session_key_filter_v1);
+  // List Filters for a Route (auth delegate_keys/owner/admin)
+  rpc list_skfs(route_skf_list_req_v1) returns (stream skf_v1);
   // List Filters for a DevAddr (auth delegate_keys/owner/admin
-  rpc get(session_key_filter_get_req_v1) returns (stream session_key_filter_v1);
+  rpc get_skfs(route_skf_get_req_v1) returns (stream skf_v1);
   // Update Filters for an Org (auth delegate_keys/owner/admin)
-  rpc update(stream session_key_filter_update_req_v1)
-      returns (session_key_filter_update_res_v1);
+  rpc update_skfs(stream route_skf_update_req_v1)
+      returns (route_skf_update_res_v1);
   // Stream Filter update (auth admin only)
-  rpc stream(session_key_filter_stream_req_v1)
-      returns (stream session_key_filter_stream_res_v1);
+  rpc stream_skfs(route_skf_stream_req_v1)
```

Comment:
we're moving the session key filters rpcs out of their own service and underneath the route service wouldn't this stream of updates just get combined with the route stream which currently outputs route, eui_pair, and devaddr_range updates and toggles based on the `oneof data` type in the `route_stream_res_v1` message?

Reply:
Done